code/_helpers/items.dm

```diff
@@ -0,0 +1,9 @@
+//Prevents robots dropping their modules.
+/proc/dropsafety(var/atom/movable/A)
+	if (istype(A.loc, /mob/living/silicon))
```

Comment:
It _was_ designed as a safety check for engiborg tools, I thought I used it in welders (tools.dm), but apparently I just used an isrobot() call.